identity/client/src/pages/groups/modals/DeleteModal.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from "react";
+import {
+  DeleteModal as Modal,
+  UseEntityModalProps,
+} from "src/components/modal";
+import useTranslate from "src/utility/localization";
+import { useApiCall } from "src/utility/api/hooks";
+import { useNotifications } from "src/components/notifications";
+import { deleteGroup, Group } from "src/utility/api/groups";
+
+const DeleteModal: FC<UseEntityModalProps<Group>> = ({
+  entity: group,
+  open,
+  onClose,
+  onSuccess,
+}) => {
+  const { t } = useTranslate();
+  const { enqueueNotification } = useNotifications();
+
+  const [callDeleteGroup, { loading }] = useApiCall(deleteGroup);
+
+  const handleSubmit = async () => {
+    if (group) {
```

Comment:
While copying the modals implementation I did change the type of `entity` to not be null, the useModal hooks also ensure that this is the case. It means that there is no need of checking `if group is not null` here or in the modals in general any more.